src/components/Map/Map.module.css

```diff
@@ -0,0 +1,8 @@
+.map {
+  position: relative;
+  height: 500px;
```

Comment:
Is there a good reason for using `px`? And not recalculating it to a `rem` value?

Reply:
No change in particular on that line aside from calculating it over to a rem value.
The HTML-root font-size is currently set to 62.5% so the value should be 500px / 10px * 1rem = `50rem`
If for some reason that seems a bit too big, try dividing by 16px instead and see if that seems more right.

Reply:
Now using `50rem` instead of `500px`. Thanks for the explanation!